### DISCUSSION TITLE
The tvl of the project is added, but the tvl field is not displayed on the page

### DISCUSSION THREAD

**Discussion by cl510804194** [DefiLlama/DefiLlama-Adapters#581] [Q&A] [answered]
I successfully added the tvl data of the flux project, but it is not displayed on https://coinmarketcap.com/currencies/flux-protocol/. Can anybody tell me why, thanks!

**Comment by 0xngmi** [comment]
cc @realdealshaman

**Comment by realdealshaman** [comment, marked as answer]
Flux is now listed and on the Defillama website. The "id" for Coingecko and CoinMarketCap has been add, not sure how often they check for this data. The TVL values should be appearing there soon.

**Reply by cl510804194** [reply]
thanks very much

**Reply by cl510804194** [reply]
I want to change Flux to Flux Protocol (FLUX),but when I submit pull request, I found that there is no place to fill in the project name，the project name on https://defillama.com/protocol/flux is Flux(FLUXK), How can I update this name?

**Reply by realdealshaman** [reply]
Hello. The changes requested have been applied and the new name will reflect on the website within 1 hour

**Reply by cl510804194** [reply]
thanks very much

**Reply by samisbakedham** [reply]
Hey, similar situation with CarthageDEX. Would you mind taking a look?
